contracts/validators.rb

```diff
@@ -0,0 +1,31 @@
+module Contracts
+  module Validators
+    def balance_valid?(balance)
+      return true unless balance.negative?
+
+      puts I18n.t('no_money')
+      false
```

Comment:
I think you don't need explicit 'false' here. puts always returns nil, which is considered like falsey result